daemon/mgr/container_utils.go

```diff
@@ -125,6 +129,23 @@ func (mgr *ContainerManager) getRuntime(runtime string) (string, error) {
 	return rPath, nil
 }
 
+// spec returns container runtime spec, unmarshal spec from config.json
+// TODO: when runtime type can be specified, it need fix
+func (mgr *ContainerManager) spec(c *Container) (*specs.Spec, error) {
```

Comment:
Could we rename this function? Actually I am afraid that we could input meaningful details in the function name, for example `GetContainerSpec`? @Ace-Tang